Util/Data.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime
+from typing import List
+from pydantic import BaseModel
+from configparser import ConfigParser
+import psycopg2
+
+
+# Database entries start
+class RawEntry(BaseModel):
+    latitude: List[float]
+    longitude: List[float]
+    image_uri: str
+    date: datetime
+
+
+class ProcessedEntry(BaseModel):
+    image_uri: str
+    plant_id: str
+# Database entries end
+
+
+def load_config(section='postgresql', filename='database.ini'):
```

Comment:
Is this the only way to configure the database connection?

Reply:
We can just run conn = psycopg2.connect(username=, password=, host=, port=), that's all this function does